sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/pipeline_with_python_function_components.ipynb

```diff
@@ -196,15 +196,17 @@
     "        training_data=input_data, max_epochs=5, learning_rate=learning_rate\n",
     "    )\n",
     "    score_with_sample_data = score_data(\n",
-    "        model_input=train_with_sample_data.outputs.model_output, test_data=test_data\n",
+    "        model_input=train_with_sample_data.outputs.model_output, test_data=test_data,\n",
```

Comment:
please take a look at the failures 
![image](https://github.com/Azure/azureml-examples/assets/47586720/a69a852f-01ac-4736-a1d1-bace21eed6c1)

Reply:
It has nothing to do with my modifications.
![image](https://github.com/Azure/azureml-examples/assets/23182548/d62f61a4-3ebb-4e70-b5e5-407a2b22d296)